modules/dnn/src/layers/resize_layer.cpp

```diff
@@ -111,7 +111,13 @@ class ResizeLayerImpl : public ResizeLayer
         internals_arr.getMatVector(internals);
 
         if (outHeight == inputs[0].size[2] && outWidth == inputs[0].size[3])
+        {
+           if (inputs[0].data != outputs[0].data)
+           {
+               inputs[0].copyTo(outputs[0]);
+           }
```

Comment:
Thank you for the fix!

1. Please fix indentation (4 spaces)
2. Please add comment that: Zero copy `outputs[0] = inputs[0]` doesn't work due to `BlobManager` (doesn't respect real refcounts from Mat).